INSTALL

```diff
@@ -98,7 +98,7 @@ additional Cartopy functionality.
 **pyepsg** 0.2.0 or later (https://github.com/rhattersley/pyepsg)
     A simple Python interface to https://epsg.io
 
-**scipy** 0.10 or later (https://www.scipy.org/)
+**SciPy** 0.10 or later (https://www.scipy.org/)
     Python package for scientific computing.
 
```

Comment:
I believe those are only at the beginning of a sentence, so I guess it depends on whether these lines are the beginning of a sentence or not.